src/Symfony/Component/Console/Tests/Helper/ProgressBarTest.php

```diff
@@ -813,10 +813,10 @@ public function testWithoutMax()
         rewind($output->getStream());
         $this->assertEquals(
             rtrim('    0 [>---------------------------]').
```

Comment:
imho it makes sense to have indentation to show continuation of string.

if we want have visual alignment for progress bar, we can do this:
```suggestion
            ''.
                rtrim('    0 [>---------------------------]').
```